tests/vmi_configuration_test.go

```diff
@@ -192,6 +192,39 @@ var _ = Describe("Configurations", func() {
 			})
 		})
 
+		Context("with EFI bootloader method", func() {
+
+			It("should use EFI without Secureboot", func() {
+				vmi := tests.NewRandomVMIWithEFIBootloader(false)
```

Comment:
This vmi doesn't have an os disk. It would be nice to be able to verify the vmi boots up successfully.

Reply:
Ah, good point. Is there a recommended VMI with a disk? I'll change the test to ensure we cover that as well.

Reply:
alpine or cirros. look for usage of the "tests.WaitUntilVMIReady" function in the tests directory. It shows to ensure the vmi both starts up and the os's console is accessible.